src/main/java/de/tum/in/www1/artemis/domain/TextBlock.java

```diff
@@ -63,10 +64,18 @@
     @JsonIgnore
     private Double addedDistance;
 
+    @OneToOne
+    @JsonIgnore
+    private Feedback feedback;
+
     @ManyToOne
     @JsonIgnore
     private TextSubmission submission;
 
+    @ManyToOne
+    @JsonIgnoreProperties("textBlocks")
```

Comment:
```suggestion
    @JsonIgnore
```
Should we exclude the assessment knowledge from the JSON? I think we would never need this outside the server?